app/src/main/java/com/iteneum/leasepert/MainActivity.kt

```diff
@@ -4,13 +4,14 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.example.ExampleView
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             LeasePertTheme {
-
+                ExampleView()
```

Comment:
If you are not going to use this function please delete it and the import as well